docs/source/technical_tutorials/search/cmr_search_in_r.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Should this be at the end, and/or does it need an explanation of what the links included here are?</p><p>Not sure why earthaccess is linked, we don't use it in this example. So basically there's no context around these links so I don't think someone would know why they might want to look at them.</p><p> </p><p>Also the CMR link should go to <a href="https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html" target="_blank">https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html</a> API docs, not the overview of what CMR is  for internal data providers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
looks good.